astropy/table/tests/test_mixin.py

```diff
@@ -1066,17 +1065,6 @@ def test_primary_data_column_gets_description():
     assert tser["a"].description == "parrot"
 
 
-def test_skycoord_with_velocity():
```

Comment:
Since we no longer have a *known* frame that uses a `DifferentialAttribute`, we cannot easily test any more that differential attributes get serialized properly (because we cannot unserialize the file).